libs/island-ui/core/src/lib/Input/Input.tsx

```diff
@@ -35,6 +35,7 @@ interface InputProps extends InputComponentProps {
   tooltip?: string
   backgroundColor?: InputBackgroundColor
   textarea?: boolean
+  type?: 'text' | 'number' | 'email' | 'tel'
```

Comment:
Unused?

Reply:
yes, it was going with the spread of ...inputProps. I moved it to the InputComponentProps and took it out of the spread